shared_example/documents_account_creation.rb

```diff
@@ -1,16 +1,21 @@
 # frozen_string_literal: true
 
-shared_examples_for 'documents_account_creation' do |type, folder_title, folder_content|
-  it "[My Documents] Check #{type} account is added to Connected Clouds Page" do
-    connected_clouds_page = my_documents_page.documents_navigation(:connected_clouds)
+shared_examples_for 'documents_account_creation' do |product, type, folder_title, folder_content|
+  it "[#{product}] Check #{type} account is added to Connected Clouds Page" do
+    case product
```

Comment:
Maybe it will be possible to store data about product type inside `TestInstance` class

So instead adding `product` parameter for shared_example we will call something like

```
case connected_clouds_page.instance.produce
when 'AppServer'
...
```

Not sure if possible and easily done, but if can be done - we can create shared_example with one lesser argument

And I think this will be actual for all `shared_examples`